src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -1006,7 +1006,7 @@ ParamBlockDesc2 g_param_block_desc(
         p_accessor, &g_pblock_accessor,
     p_end,
 
-    ParamIdGlobalBounceLimit, L"global_illumination_bounces", TYPE_INT, P_TRANSIENT, 0,
+    ParamIdGlobalBounceLimit, L"max_ray_bounces", TYPE_INT, P_TRANSIENT, 0,
```

Comment:
Are we breaking any form of backward compatibility with this change?

Reply:
I haven't had issues loading older scenes but I'm not 100% sure. I will revert it back just to be on the safe side.